### PR TITLE
Text Aligned to Start & Dependencies Updated

### LINKED ISSUE #69: Text alignment in Forum Edit text
Currently the edit text in forumn aligns text centrally, but it doesn't look, so the text inside the edit text should start from left.
The file to be changed is activity_forum.xml.

### PR DESCRIPTION
**Pull request by Pranav915**
Fixes #69 
## Description

Fix #69

1) Text inside EditText of Forum is Aligned to Start.
2) Dependencies in the Gradle files are updated.
## Checklist:

- [x] I used the Reformat code option in android studio for the files I changed
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] The App is running Fine and as Expected In my Emulator or Physical Device
- [x] I have checked my code and corrected any misspellings
- [x] My changes didn't cause any issues for the other functionalities present before.
- [x] This PR is created From a branch which is upto date with the master branch

### CODE REVIEW

**Review by ash-100 — COMMENTED** [review]
do not give any fixed values to layout width or height

**Comment by Pranav915** [comment]
> do not give any fixed values to layout width or height

Ok

File: app/src/main/res/layout/activity_forumn.xml [hunk 1]
```diff
@@ -50,10 +48,11 @@
         android:layout_width="wrap_content"
         android:layout_height="40dp"
         android:background="@color/transparent"
+        android:contentDescription="@string/forum_send_button"
         android:src="@drawable/send_ic"
         app:layout_constraintBottom_toBottomOf="@id/message_input"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@id/message_input"
+        app:layout_constraintStart_toEndOf="@+id/chat_recycler"
```
Comment (Amar-2003) [inline app/src/main/res/layout/activity_forumn.xml:55]: This doesn't seem right to me, it should be message_input itself

**Review by Amar-2003 — CHANGES_REQUESTED** [review]
PTAL

**Comment by Amar-2003** [comment]
I have fixed the merge conflict

**Review by Amar-2003 — APPROVED** [review]
LGTM!

**Review by ash-100 — APPROVED** [review]

**Pull request merged**